src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -45,6 +44,16 @@ public static async Task<FunctionMetadataResponse> ToFunctionMetadataResponse(th
                 InvokeUrlTemplate = GetFunctionInvokeUrlTemplate(baseUrl, functionMetadata, routePrefix)
             };
 
+            if (!string.IsNullOrEmpty(functionPath))
+            {
+                response.ScriptRootPathHref = VirtualFileSystem.FilePathToVfsUri(functionPath, baseUrl, hostOptions, isDirectory: true);
+            }
+
+            if (!string.IsNullOrEmpty(functionMetadataFilePath))
+            {
+                response.ConfigHref = VirtualFileSystem.FilePathToVfsUri(functionMetadataFilePath, baseUrl, hostOptions);
```

Comment:
What are these hrefs used for? Are they okay to be null?

Reply:
Unless something changed since the initial design, those should be OK as null

Reply:
These APIs are used by Portal to get the files for "Code + Test". Right now, it generates a VFS link that's invalid and throws error when called. So, I am making these nulls and working with Portal so they can handle the values.

I will confirm with Portal before merging, but this shouldn't have a negative impact.